File: fabric/src/main/java/com/rpmtw/rpmtw_platform_mod/fabric/RPMTWPlatformModFabric.java
package com.rpmtw.rpmtw_platform_mod.fabric;

import com.rpmtw.rpmtw_platform_mod.RPMTWPlatformMod;
import net.fabricmc.api.ModInitializer;

public class RPMTWPlatformModFabric implements ModInitializer {
    @Override
    public void onInitialize() {
        RPMTWPlatformMod.init();
    }
}

File: fabric/src/main/kotlin/com/rpmtw/rpmtw_platform_mod/fabric/RPMTWPlatformModFabric.kt
package com.rpmtw.rpmtw_platform_mod.fabric

import com.rpmtw.rpmtw_platform_mod.fabriclike.RPMTWPlatformModFabricLike
import net.fabricmc.api.ModInitializer

class RPMTWPlatformModFabric : ModInitializer {
    override fun onInitialize() {
        RPMTWPlatformModFabricLike.init()
    }
}